jwst/tweakreg/astrometric_utils.py

```diff
@@ -96,8 +96,10 @@ def create_astrometric_catalog(input_models, catalog="GAIADR3", output="ref_cat.
         else Time(input_models[0].meta.observation.date).decimalyear
     )
     ref_dict = get_catalog(fiducial[0], fiducial[1], epoch=epoch, sr=radius, catalog=catalog)
+    if len(ref_dict) == 0:
+        return ref_dict
```

Comment:
I assume the calling routine, to which this is being returned, is already setup to handle the case where `ref_dict` has zero length?

Reply:
yes, that is correct